buku

```diff
@@ -2865,50 +2865,31 @@ class BukuDb:
             True on success, False on failure.
         """
 
-        ff_bm_db_paths = {}
-        firefox_profile = firefox_profile and [firefox_profile]
-
         if sys.platform.startswith(('linux', 'freebsd', 'openbsd')):
             gc_bm_db_path = '~/.config/google-chrome/Default/Bookmarks'
             cb_bm_db_path = '~/.config/chromium/Default/Bookmarks'
             vi_bm_db_path = '~/.config/vivaldi/Default/Bookmarks'
-
-            default_ff_folder = os.path.expanduser('~/.mozilla/firefox')
-            profiles = firefox_profile or get_firefox_profile_names(default_ff_folder)
-            if profiles:
-                ff_bm_db_paths = {s: '~/.mozilla/firefox/{}/places.sqlite'.format(s)
-                                  for s in profiles}
```

Comment:
This got moved into a separate function, since it's exactly the same processing other than the prefix.
(Also it's easier to test for correctness this way.)